src/USER-MISC/fix_pafi.h

```diff
@@ -0,0 +1,101 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef FIX_CLASS
+
+FixStyle(pafi,FixPAFI)
+
+#else
+
+#ifndef LMP_FIX_PAFI_H
+#define LMP_FIX_PAFI_H
+
+#include "fix.h"
+#include "compute.h"
```

Comment:
there is no need to include "compute.h" here. just use a forward declaration like with RanMars.